specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json

```diff
@@ -779,6 +779,18 @@
             "type": "string"
           },
           "type": "object"
+        },
+        "managedVirtualNetwork": {
+          "description": "Workspace managed virtual network",
+          "type": "string"
+        },
```

Comment:
what is this?  I haven't seen this in other private link reviews

Reply:
> what is this? I haven't seen this in other private link reviews

This property is returned when creating workspace. Please see the ./examples/CreateOrUpdateWorkspace.json

Reply:
That doesn't answer the question about what the property means.  Is it read only?  What values can it have?  What does it do?  The description and other context in the swagger spec does not provide adequate information.

Reply:
> That doesn't answer the question about what the property means. Is it read only? What values can it have? What does it do? The description and other context in the swagger spec does not provide adequate information.

This is a readonly property mandated by NRP for a workspace that supports private endpoints.  The value can be "default".

Reply:
The description should be updated to indicate what effect this property has and when/why someone should be setting it.  The description right now is just a non-camel cased variant of the property name.

Also, are you sure this shouldn't be a resource ID of a virtual network?  How are you referencing a virtual network by name?  VNet names are not unique by themselves.

Reply:
> The description should be updated to indicate what effect this property has and when/why someone should be setting it. The description right now is just a non-camel cased variant of the property name.
> 
> Also, are you sure this shouldn't be a resource ID of a virtual network? How are you referencing a virtual network by name? VNet names are not unique by themselves.

We are sure that this shouldn't be a resource ID of a virtual network. Now this is a read only property return by NRP. Now the value can only be default but we will support more value later so we can't add default value here. It should be string.

Reply:
> The description should be updated to indicate what effect this property has and when/why someone should be setting it. The description right now is just a non-camel cased variant of the property name.
> 
> Also, are you sure this shouldn't be a resource ID of a virtual network? How are you referencing a virtual network by name? VNet names are not unique by themselves.

Setting this to 'default' will ensure that all compute for this workspace is in a virtual network managed on behalf of the user

Reply:
The description for this property still needs to be updated.  If its readonly as you mention above please mark it as readonly in swagger.

Reply:
> The description for this property still needs to be updated. How does a user know any of the information that you shared in this thread? If its readonly as you mention above please mark it as readonly in swagger.

Hi @pilor , 
"Setting this to 'default' will ensure that all compute for this workspace is in a virtual network managed on behalf of the user" This description is from backend team, can I use it directly?
Or could you please provide a good description? 

I will mark it as readonly but for the description I have no idea. This is not a block issue. I will update it again with the original description from backend team.

Reply:
> The description for this property still needs to be updated. How does a user know any of the information that you shared in this thread? If its readonly as you mention above please mark it as readonly in swagger.

I will confirm with backend team to confirm whether it is readonly or not again. Sorry

Reply:
> The description for this property still needs to be updated. How does a user know any of the information that you shared in this thread? If its readonly as you mention above please mark it as readonly in swagger.

Hi, This property is not readonly.  I am so sorry for previous wrong comment.

Reply:
I have updated the description to "Setting this to 'default' will ensure that all compute for this workspace is in a virtual network managed on behalf of the user."
This description is provided by backend team. I think that it is correct and make sense.
The only question is that maybe you will ask why not add default value here. 
use "default" to indicate what the server does if the value is not provided by customers. But we ask customer to provide this parameter.